### ISSUE TITLE
Make Hack card a concrete example

### ISSUE THREAD

**Issue by johnanvik** [SibylLab/Program-Wars#455]
> The hack card is not related to programming concepts. Its idea is interesting as a game, but the concept may be better defined. Could it be like an SQL injection or other program attack technique? I mean, to have a concrete context. (SIGCSE Reviewer #5)

**Comment by johnanvik** [comment]
I think this would look like making "hack" a card type and then having cards with different names and images.

**Comment by groadybones** [comment]
Would the hacks do different things depending on the kind?

**Comment by johnanvik** [comment]
I don't think so -- at least not for now. We can consider this later on when we come up with the "new" cards and see if that makes sense at that time.

**Comment by johnanvik** [comment]
@strinsberg proposed the following refinement in the same spirit as the refined malware cards (i.e. #534).

```
 **Bot net** - subtracts 10 points from the affected player and gives them to the opponent who played it. This is the effect you had for ransomware. The points are returned if the card is removed.

**Hack** - no longer removes stacks as this is basically what virus does. Messes up all scores for the active player by adding or subtracting a random number up to 20. Originally the effect for trojan.

**DDOS** - similar to the proposed poweroutage cards. While affected a player cannot play any cards that would end up on their play field. Only lasts 2 turns.

**Backdoor** - Just something I thought of. If the affected player has antivirus this card will allow the player who played it (only that player) to use virus cards on the affected player. Sort of just bypasses the antivirus.
```

**Comment by johnanvik** [comment]
Will be addressed by #602.

**Comment by johnanvik** [comment]
Fixed in PR #618
